config/settings.yml

```diff
@@ -1744,7 +1744,11 @@ ogc:
     api_key: fake_key
 
 accredited_representative_portal:
-  pilot_users_email_poa_codes: ~
+  allow_list:
+    github:
+      access_token: some_fake_token
+      repo: faked/fakeness
+      path: not_real.csv
```

Comment:
you will add these to the manifest repo and AWS parameter store if needed?

Reply:
@rmtolmach Are manifest repo and AWS parameter store the overriding source for `Settings`? Would that be a smart idea for this use case rather than some checked-in `settings.yml` file for production? The repo itself is not public. I can definitely ask around for more platform security guidance here!

Reply:
@rmtolmach I changed it to the squiggle so it will get the value from wherever the source is.

Reply:
> Are manifest repo and AWS parameter store the overriding source for `Settings`?

Yes. The settings are in a state of transition at the moment, but you can [follow the instructions here](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-VetsAPIsettingsandsecrets) to set up those variables for now. This will likely change in the next couple weeks, so if you don't need this all wired together quite yet, it might be smart to wait.